### ISSUE TITLE
error running ember install ember-cli-bootswatch

### ISSUE THREAD

**Issue by lmcardle** [Panman8201/ember-cli-bootswatch#10]
I get the below error when trying to install ember-cli-bootswatch:

```
$ember install ember-cli-bootswatch
version: 0.2.3
Installed packages for tooling via npm.
installing
Installing browser packages via Bower...
  not-cached git://github.com/twbs/bootstrap.git#^3.3.1
  not-cached git://github.com/thomaspark/bootswatch.git#^3.3.1
  resolved git://github.com/twbs/bootstrap.git#3.3.4
  resolved git://github.com/thomaspark/bootswatch.git#3.3.4+1
Installed browser packages via Bower.
installing
The `ember generate` command requires an entity name to be specified. For more details, use `ember help`.
```

**Comment by Panman8201** [comment]
Must be a recent change for addons, I'll investigate. Thanks for the report.

**Comment by Panman8201** [comment]
Oh, and just to note, this is specifically for the bower stuff. So, as long as you have `bootswatch` installed in bower, this addon should still work.

**Comment by Panman8201** [comment]
Looks like this was fixed with ember-cli 0.2.4. Tried a quick test  with 0.2.5 and everything seemed to install without error.
